carto/kuvizs.py

```diff
@@ -0,0 +1,77 @@
+"""
+Module for working with map custom visualizations (aka Kuvizs)
+
+.. module:: carto.Kuvizs
+   :platform: Unix, Windows
+   :synopsis: Module for working with map custom visualizations (aka Kuvizs)
+
+.. moduleauthor:: Simon Martin <simon@carto.com>
+
+"""
+
+import base64
+
+from pyrestcli.fields import CharField, DateTimeField
+
+from .resources import Manager, WarnResource
+from .paginators import CartoPaginator
+
+
+API_VERSION = "v4"
+API_ENDPOINT = "api/{api_version}/kuviz/"
+
+PRIVACY_PASSWORD = 'password'
+
+
+class Kuviz(WarnResource):
+    """
+    Represents a map custom visualization in CARTO.
```

Comment:
Let's add a warning like in other cases:

```
    .. warning:: Non-public API. It may change with no previous notice
```

Reply:
It should a public one

Reply:
For the records `public == documented in developer center && versioned`

Reply:
Ok, I am going to add the message